src/backend/gporca/data/dxl/minidump/CTAS-random-distributed-from-replicated-distributed-table.mdp

```diff
@@ -305,7 +305,7 @@
               <dxl:SortingColumnList/>
               <dxl:TableScan>
                 <dxl:Properties>
-                  <dxl:Cost StartupCost="0" TotalCost="431.000023" Rows="3.000000" Width="12"/>
+                  <dxl:Cost StartupCost="0" TotalCost="431.000023" Rows="1.000000" Width="12"/>
```

Comment:
I found this plan very interesting. This node executes on 3 segments in fact (see plan in the header). Existing explain logic divides this value by 3.

Reply:
So what? Please explain why is changes of DXL and resulting plan valid?

```diff
+                     ->  Seq Scan on test_replicated  (cost=0.00..431.00 rows=2 width=6)
-                     ->  Seq Scan on test_replicated  (cost=0.00..431.00 rows=4 width=6)
```